### PR TITLE
all: gofmt

### PR DESCRIPTION
**Pull request by whilei**
Run 'gofmt -w .' from project root.

* [n/a] Updated all relevant documentation in docs
* [n/a] Updated all code comments where relevant
* [n/a] Wrote tests
* [x] Updated CHANGELOG.md

### CODE REVIEW

**Comment by whilei** [comment]
@xla Hm, interesting... what do you mean generally? Is there an ambiguity in `gofmt`?

**Comment by xla** [comment]
@whilei I'm not talking about the changes in this PR and that the order of the imports is not always correct.

**Comment by whilei** [comment]
Can you point to an example?

As far as I see, these proposed changes only alphabetize a few _internal packages_ sections.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+* June 13, 2018
+
+SYNTAX
```
Comment (melekes) [inline CHANGELOG.md]: No need in the changelog entry for such changes. Let 's spare the time of the people who read it.

**Comment by xla** [comment]
@whilei Thanks for your contribution! Ultimately we want to have even more consistency for the imports, which is out-of-scope for this PR.

**Pull request merged**